### ISSUE TITLE
Enhancement request: Leverage additional tags to disambiguate different releases of the same album

### ISSUE THREAD

**Issue by audiomuze** [LMS-Community/slimserver#878]
Whilst LMS treats each folder as a distinct album and this avoids blending multiple versions of an album into a single album of repeated tracks, it does not provide any visual means of disambiguating different releases of an album from others.  

Musicbrainz uses a a release MBID to differentiate one release of an an album from another. This is hugely beneficial when one has numerous releases of an album e.g. the original release and subsequent remasters or deluxe editions, reissues etc. or a redbook and high resolution release.  It lends context to the likes of this (different releases of the same album, where LMS currently provides no information to be able to distinguish one from another):

![image](https://user-images.githubusercontent.com/6852511/230791359-67aacb21-192e-4c77-9c64-9cb8b90cc3d4.png)

If LMS leveraged releaseid (for those that have it), or better yet a user defined tag e.g. VERSION and appended same to the album title when displayed it'd make it easy to distinguish one release from another.

Ideally LMS server settings would enable mapping of user nominated tags to a field that LMS uses internally, making for maximum flexibility for users e.g. catalognumber, version,  etc.

**Comment by dingoagogo** [comment]
> Whilst LMS treats each folder as a distinct album and this avoids blending multiple versions of an album into a single album of repeated tracks

I don't think this is correct. My LMS library is a bit of a mess, and this has been a big part of the problem. Every album is in its own folder - some even on different drives, however LMS nearly always groups different releases or versions together as a single album with repeated tracks. I just discovered that the few albums this doesn't happen to had one thing in common: They were older mp3 tracks I had tagged using Musicbrainz Picard. While they contained identical information in the basic tags (Album, AlbumArtist, Year, etc.) as their FLAC counterparts - and would therefore be expected to show up with the other files as a single album, LMS separated them into their own album. In contrast, I have not been using Picard for tagging my newer FLAC additions so far.

As a test, I created a new LMS media library. Inside the base music folder I put two folders. Into each of these folders I copied an identical file - a single FLAC with some basic tags, but no Musicbrainz tags. LMS listed the two imported tracks under one album. I then edited the tags on these files, adding only a MUSICBRAINZ_ALBUMID tag. I gave one a value of "1", and the other a value of "2." I cleared the library and rescanned. This time, LMS used the added Musicbrainz tags to disambiguate the albums as separate, and each containing one track.

This revelation alone is going to make a huge difference in the usability of my library once I add Musicbrainz tags to my FLAC files. **However, @audiomuze's point of leveraging additional tags to provide a visual indication of which version (etc.) of an album is which while browsing would be greatly appreciated!** As they said, simply choosing a few tags to achieve this goal would be ideal.

As potentially another option - and I'm sure this would be more involved - I for one would like to be able to choose my own tags to append in strings and create my own "Virtual" tags to sort and group with.

Last point, (semi-related, but perhaps this should be its own issue:) In addition to the tags @audiomuze has suggested, I think utilizing the DISCSUBTITLE tag would be very helpful to subdivide tracks inside an album, for instance when the discs of a multi-disc album are individually named, or in the case of Bonus Tracks, or when a release contains both a Mono and Stereo version on a single disc.

**Comment by audiomuze** [comment]
Implementing the suggestion above will enable the above presentation to be turned into something akin to this:

![Screenshot 2023-06-18 161349](https://github.com/Logitech/slimserver/assets/6852511/8d542543-f622-435b-b70f-6a86afdf42f0)

Thoughts from others?

**Comment by mundschenk-at** [comment]
Currently I add information in square brackets to album titles differentiate releases, e.g. `Some Album [1992 remaster]`. I also use this "editorial annotation" to add extra information like `[live]` to song titles. Having defined tags instead (and some visual representation in the UI) would be beneficial for doing automated cleanup of tagging information (i.e. [bliss](https://blisshq.com) does not know about my annotation style).

**Comment by audiomuze** [comment]
> Currently I add information in square brackets to album titles differentiate releases, e.g. `Some Album [1992 remaster]`. I also use this "editorial annotation" to add extra information like `[live]` to song titles. Having defined tags instead (and some visual representation in the UI) would be beneficial for doing automated cleanup of tagging information (i.e. [bliss](https://blisshq.com) does not know about my annotation style).

Agreed, it will also make it a lot easier for LMS and other servers to find all performances of a song/composition and present that in one place. All song notations like Live, Acoustic, Demo, Instrumental actually belong in a tag called SUBTITLE.

**Comment by audiomuze** [comment]
Just bumping this, it'd be great not to have to stuff album names with album resolution or catalog number etc. to stop LMS conflating two or more versions of the same album.

**Comment by allmazz** [comment]
It would be great, it's the only thing stopping me from migrating from Roon
